types/src/executable.rs

```diff
@@ -15,12 +15,15 @@ pub enum ExecutableDescriptor {
 }
 
 pub trait ModulePath {
-    // TODO(George):
+    // TODO(loader_v2):
     //   Improve this in the future, right now all writes use state keys
```

Comment:
supernit / OCD : comment can fit in 2 lines :)

Reply:
I keep doing 80 lines and forget that our default is 100😄. FIxed.